public/docs/ts/latest/guide/upgrade.jade

```diff
@@ -580,20 +572,21 @@ figure
   A simple example of an upgradable component is one that just has a template
   and a controller:
 
-+makeExample('upgrade-adapter/ts/app/upgrade-static/hero-detail.component.ts', null, 'hero-detail.component.ts')
++makeExample('upgrade-module/ts/app/upgrade-static/hero-detail.component.ts', 'hero-detail', 'hero-detail.component.ts')
 
 :marked
-  We can *upgrade* this component to Angular 2 using the `UpgradeAdapter`'s
-  `upgradeNg1Component` method. It takes the name of an Angular 1 component
-  directive and returns an Angular 2 **component class**. 
-  Declare it in an `NgModule` as with other  Angular 2 components:
+  We can *upgrade* this component to Angular 2 using the `UpgradeComponent` class. 
+  By creating a new Angular 2 **directive** that extends `UpgradeComponent` and doing a `super` call
+  inside it's constructor, we have a fully upgrade Angular 1 component to be used inside Angular 2.
+  All that is left is to add it to `AppModule`'s `declarations` array.
 
-+makeExample('upgrade-adapter/ts/app/upgrade-static/upgrade_adapter.ts', 'heroupgrade', 'app.module.ts')
++makeExample('upgrade-module/ts/app/upgrade-static/hero-detail.component.ts', 'hero-detail-upgrade', 'hero-detail.component.ts')
++makeExample('upgrade-module/ts/app/upgrade-static/app.module.ts', 'hero-detail-upgrade', 'hero-detail.component.ts')
 
 .alert.is-helpful
   :marked
-    Upgraded components always have an element selector, which is based
-    on the original name of the original Angular 1 component directive.
+    Upgraded componentes are Angular 2 **directives**, instead of **components**, because Angular 2
+    is unaware that Angular 1 will create elements under it.
```

Comment:
I am not sure if I understand the "is unaware that Angular 1 will create elements under it."